docs/users/overview.adoc

```diff
@@ -135,3 +135,12 @@ The example of Process Data for different tasks can be as follows.
 ** RPA robot data
 ** external API call
 ** ...
+
+=== Overlays
+
+`bpmn-visualization` provides https://process-analytics.github.io/bpmn-visualization-js/api/classes/bpmnelementsregistry.html#addoverlays[API to add overlays] on top of BPMN elements. This additional overlays layer permits you to show whatever data you want and enrich your diagrams.
+
+==== Overlays examples
+To see overlays simple examples, go to the https://cdn.statically.io/gh/process-analytics/bpmn-visualization-examples/master/examples/index.html#overlays[**icon:forward[] Simple examples - live demo**] You will find out how to add overlays to BPMN elements and how to apply style to them.
```

Comment:
⚠️ I suggest we put a gif or screenshot here from the demo to illustrate what we are talking about.